ores/ores.py

```diff
@@ -30,7 +28,7 @@ def main():
         sys.stderr.write(USAGE)
         sys.exit(1)
     elif sys.argv[1] in ("-h", "--help"):
-        sys.stderr.write(__doc__ + "\n")
+        sys.stderr.write(__doc__ % { "usage": USAGE })
```

Comment:
Yes, user error.

I ran `./utility -h`. It told me to run `ores dev_server`. So I did `./utility ores dev_server` :)